scripts/filters/delete-mask-handle.js

```
@@ -0,0 +1,27 @@
+/* delete mask */
+
+"use strict";
+
+hexo.extend.filter.register(
+  "after_post_render",
+  function (data) {
+    const theme = this.theme;
+
+    // 处理del标签的代码
+    const regPureDelTag = /<del(?:\s+[^>]*)?>((?:(?!<\/?del[\s>])[^])*)<\/del>/g;
```

Biome (1.9.4)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)